src/transformStateWithClones.js

```diff
@@ -1,13 +1,37 @@
 'use strict';
 
+const { meta: action } = require('eslint-plugin-node/lib/rules/no-sync');
```

Comment:
The import statement `const { meta: action } = require('eslint-plugin-node/lib/rules/no-sync');` is not used in the code. Consider removing it to clean up the code.